config/application.rb

```diff
@@ -24,7 +24,7 @@ class Application < Rails::Application
 
     # I18n stuff
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}')]
-    config.i18n.available_locales = [:en, :es, :fr, :pl]
+    config.i18n.available_locales = [:en, :es, :fr, :pl, :"pt-BR"]
```

Comment:
I'm actually not 100% sure that would make the merge conflict go away, since it's still a modification to the same line of the same file. I think Git will still see it as a merge conflict. Sorry about that.

If the suggested change doesn't get rid of the merge conflict, we will handle it during the merge.

Reply:
I can confirm that suggestion doesn't get rid of the merge conflict.

So it's safe to disregard these comments.